Update and rename _open-spaces.mdx to open-spaces.mdx (#1762)
EuroPython/website: src/content/pages/open-spaces.mdx

---
title: EuroPython 2026 Open Spaces
url: /open-spaces
subtitle: Organise or join an open space at EuroPython 2026!
---

> # EuroPython 2026 Open Spaces

Reviewer: Could leave "EuroPython 2026" out of the title, this is the EuroPython 2026 website :)

```suggestion
# Open Spaces
```

add their session idea to the Open Spaces registration calendar, which also
serves to attract potential participants interested in joining.

> Check out the [schedule](https://ep2026.europython.eu/schedule/) to see the Open Spaces!

Reviewer 2: I think relative path is preferred, so it should be: 
`Check out the [schedule](/schedule/) to see the Open Spaces!`

But I am a bit confused here. On the schedule, it is not clear which ones are for open spaces. Will it be added to the conference main schedule? If so, does it make sense to link /schedule/#day-2026-07-15 instead of tutorial days?

Author: Hi! Since no open spaces have been reserved yet, the rooms don't appear on the schedule. Once one is reserved, that room will appear there :) And yes, it makes much more sense to use /schedule/#day-2026-07-15, but this gives me a File not found error, am I doing it wrong?

Reviewer 2: I see! Thanks for the explanation! 

So on localhost, I did `[schedule](/schedule#day-2026-07-15)` and it seems to render. And it looks good on preview, I think. It does not for you?

Author: It's in my side for sure then, I'm not used to use github 😅 if it works for you then it should be fine!

Reviewer 2: hey hey! I am a little bit scared of what I've missed now :D Would you mind double checking the preview here: https://ep-website--1762.org.readthedocs.build/open-spaces/ to see if the schedule links are as expected? Just in case I was tripping.

Author: It works perfectly!

Reviewer 2: Woo! Thank you so much! Then feel free to merge whenever you feel comfortable. If you add more stuff, happy to do another round of review any time! <3

> Check out the [schedule](https://ep2026.europython.eu/schedule/) to see the Open Spaces!

Reviewer: Let's prefer relative links:
```suggestion
Check out the [schedule](/schedule/) to see the Open Spaces!
```

## How to Organise an Open Space

To organise your own Open Space session, simply book a free time slot and room
via our

> [Open Spaces registration calendar](https://calendar.google.com/calendar/u/0?cid=Y18zYzU5ODY4ZTFiY2Q5MmM4OWFhNjc2YTQ0M2FhNDFhOGIxNjk0YzY3MzVlNmY0MzZkNGU4MDdiNTk0ZjA5MjlkQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)

Reviewer 2: I actually got really confused about this last year too. So to clarify, this "registration calendar" link cannot be used directly to book a session, right? Is this a view-only calendar to show what has been booked? If we want to book a session, we have to use one of the 3 links below in line 38-39, right?

Author: You are completely right, this was carried over from last year, but that's no longer the case. Fixed!

Reviewer 2: Thank you!

— this allows you to easily secure a space to host your discussion or

> presentation. Once booked, your session will be listed on the [schedule](https://ep2026.europython.eu/schedule/) shortly.

Reviewer: ```suggestion
presentation. Once booked, your session will be listed on the [schedule](/schedule/) shortly.
```

> presentation. Once booked, your session will be listed on the [schedule](https://ep2026.europython.eu/schedule/) shortly.

Reviewer 2: same as https://github.com/EuroPython/website/pull/1762/changes#r3551445572

Reviewer 2: This looks good too!

Rooms S4(1,2,3), S4(4), and S4(5) will be reserved exclusively for Open Space
sessions. You can book one of these spaces by using the links below:

[Room S4(1,2,3)](https://calendar.app.google/t7MHRuVLYRkJXtAK7) | [Room S4(4)](https://calendar.app.google/Phw2NBXTyqjj8Lwt6)
| [Room S4(5)](https://calendar.app.google/JjtSSNgxnxeUTASt7)

Please note that:
- Room S4(1,2,3) can accommodate 48 people and has a 85'' monitor
- Room S4(4) can accommodate 24 people and has no monitor

> - Room S4(5) can accommodate 36 people and has a 85'' monitor

Reviewer: ```suggestion
- Room S4(1,2,3) can accommodate 48 people and has an 85" monitor
- Room S4(4) can accommodate 24 people and has no monitor
- Room S4(5) can accommodate 36 people and has an 85" monitor
```
